examples/99-advanced/sphere_eversion.py

```diff
@@ -4,30 +4,20 @@
 Turning the sphere inside out
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There are several videos online talking about how a sphere can
-be turned inside out in a continuous fashion, for instance in
-`this YouTube video <https://www.youtube.com/watch?v=OI-To1eUtuU>`_.
-Thanks to |the paper| (also available
-`on arXiv <https://arxiv.org/abs/1711.10466>`_), we can plot this
-so-called eversion of a sphere (turning it inside out without
-pinching or tearing the surface, in other words by preserving its
-topology).
-
-The mathematics involved can seem a bit, well, involved. What
-matters is the overall process visible in the animation: first
-the sphere is corrugated and stretched out a bit to allow some
-legroom for the smooth transformation, then the lobes are twisted
-around through each other, and the process is reversed in order
-to unfold the sphere. It's not obvious that the transformation
-is truly smooth; this was proved in the paper by Bednorz and
-Bednorz!
-
-.. |the paper| raw:: html
-
-   <!-- this hack is necessary to have boldface in a hyperlink -->
-   <a href=https://doi.org/10.1016/j.difgeo.2019.02.004>an excellent
-   paper by Adam Bednorz and Witold Bednorz, Differential Geometry
-   and its Applications <b>64</b>, 59 (2019)</a>
+There are several videos online talking about how a sphere can be turned inside
+out in a continuous fashion, for instance in `this YouTube video
+<https://www.youtube.com/watch?v=OI-To1eUtuU>`_.  Thanks to `the paper
```

Comment:
I noticed a warning about the paper reference earlier, but did that actually break CI, or just lead to noise? I thought it didn't cause an issue in the earlier PR I noticed it in...

Anyway, I'd prefer to keep the actual text of the link if that's possible (sans boldface, obviously): `the paper` -> `an excellent paper by Adam Bednorz and Witold Bednorz, Differential and its Applications 64, 59 (2019)`.

Reply:
![tmp](https://user-images.githubusercontent.com/11981631/216166105-e27e4245-6f05-4829-b230-6de58d9cc0f7.png)

Reply:
Thanks. I actually posted a follow-up comment but it was stuck in review land :angry: The build looked fine in the hydrogen orbitals example PR build:

![Screenshot from 2023-02-01 22-10-49](https://user-images.githubusercontent.com/17914410/216164147-21cf4110-b856-459b-a086-111183c36852.png)

So it seemed to work fine. Oh well.